python/lsst/obs/pfs/constructNormFlatTask.py

```diff
@@ -95,8 +90,8 @@ def combine(self, cache, struct):
         for expRef in dataRefList:
             exposure = expRef.get('postISRCCD')
             xOffsets.append(exposure.getMetadata().get('sim.slit.xoffset'))
```

Comment:
This looks like a keyword from Craig's simulator.  How will this work with real data?